dashboards/AdminRouter/AdminRouter-Details-Runtime.json

```diff
@@ -1075,7 +1068,7 @@
     ]
   },
   "timezone": "",
-  "title": "Admin Router: Details Runtime",
+  "title": "/AdminRouter/AdminRouter-Details-Runtime",
```

Comment:
Looks like all the dashboard titles got reverted, can we change them back to the old format? Thanks!

Reply:
Whenever the dashboards specified in the beta-dcos-monitoring package are automatically imported, the titles get converted.
How do I prevent that? It also fks up linking of graphs every time^^

Reply:
This has been fixed in dcos-monitoring, but it won't be released till the GA release, so unfortunately the beta will continue to do this :(